docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/docker-apps-inner-loop-workflow.md

```diff
@@ -114,7 +114,7 @@ In the DockerFile, you also need to instruct Docker to listen to the TCP port th
 
 There are other lines of configuration that you can add in the DockerFile depending on the language/framework you are using, so Docker knows how to run the app. For instance, you need the ENTRYPOINT line with \["dotnet", "MyCustomMicroservice.dll"\] to run a .NET Core app, although you can have multiple variants depending on the approach to build and run your service. If you're using the SDK and dotnet CLI to build and run the .NET app, it would be slightly different. The bottom line is that the ENTRYPOINT line plus additional lines will be different depending on the language/platform you choose for your application.
 
-**More info** For information about building Docker images for .NET Core applications, go to <https://docs.microsoft.com/dotnet/core/docker/building-net-docker-images>.
+**More info** For information about building Docker images for .NET Core applications, go to <../../../core/docker/building-net-docker-images.md>.
```

Comment:
This change should be reverted. The use of \<…> for a link means that that link text and the link URL should be the same. However, this isn't a valid URL.